### PR TITLE
Task social card

### PR DESCRIPTION
**Pull request by ekachernova**
Updated pull request

### CODE REVIEW

File: css-basics_social-card/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+* {
```
Comment (VanessaSchwab1906) [inline css-basics_social-card/css/styles.css:1]: you could add a font-family but not necessary for the exercise :)

File: css-basics_social-card/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  box-sizing: border-box;
+  margin: 10px;
+}
+body {
+  background-color: whitesmoke;
+}
+.card-wrapper {
```
Comment (VanessaSchwab1906) [inline css-basics_social-card/css/styles.css:8]: I think you could also use "article" as selector

File: css-basics_social-card/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  width: 800px;
+  background-color: white;
+  border-radius: 12px;
+  box-shadow: 12px;
+}
+img {
+  border-radius: 12px;
+}
```
Comment (VanessaSchwab1906) [inline css-basics_social-card/css/styles.css:16]: you could also add some styling to "h2"(f.e. text-transform, font-size) and "p" (f.e. font-size, margin)

**Review by VanessaSchwab1906 — COMMENTED** [review]
everythings good, just added some some extra hints :)

File: css-basics_social-card/css/styles.css (hunk 2 above)
Comment (ekachernova) [reply css-basics_social-card/css/styles.css:8]: I will do that

**Pull request merged**